src/app/containers/Ad/Canonical/index.jsx

```diff
@@ -1,6 +1,16 @@
 import React from 'react';
+import { Helmet } from 'react-helmet';
 import AdSlot from './AdSlot';
 
-const CanonicalAd = () => <AdSlot uniqueId="leaderboard" />;
+const CanonicalAd = () => (
+  <>
+    {/* Loading dotcom-bootstrap.js here instead of CanonicalAdBootstrapJs to avoid it loading on live */}
```

Comment:
I don't think we need to do this, because in the file below https://github.com/bbc/simorgh/pull/6814/files#diff-2a790056302ef5d6fa004eb0da022458R87 we check for `hasAds` before adding the CanonicalAdBootstrapJs and so long as we have `hasAds` set to false, this should be fine.

Reply:
I moved it here temporarily because the logic in FrontPage does those, but it doesn't check the toggles. Which means it will run on live. The small snippet should be fine, but I figure we don't quite trust the GNL script yet for live? And we're pointing at its test version.

Reply:
Would checking the toggle value in CanonicalAdBootstrapJs be an option so we don't need to move this code here temporarily? And ads are looking good when testing locally :D